website/components/blog/PostsList.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+import PostPreview, { Post } from "./PostPreview"
+
+export interface Props {
+  title: string
+  posts: Array<Post>
+}
+
+const PostsList: React.FC<Props> = ({ title, posts }) => {
+  return (
+    <>
+      <h2>{title}</h2>
+      <main className="row">
+        {posts.map((post, index) => {
+          const isLarge = index < 2
```

Comment:
it's always good to name this kind of "magic" numbers. They can be named inside the component, in some config file, or in .env. Here it would probably be something like `numberOfLargePosts` or `numberOfEmphasizedPosts` etc. 

It's easier to understand what this number is via name than via logic